src/data_warehouse/cli/ingestion.py

```
@@ -0,0 +1,71 @@
+"""Ingestion-related CLI commands."""
+
+import click
+from loguru import logger
+
+
+@click.group()
+def ingestion():
+    """Data ingestion commands.
+
+    These commands allow you to ingest data from various sources into the data warehouse.
```

Ruff (0.8.2)

Line too long (89 > 88)

(E501)